modules/core/src/v2/wallet.ts

```diff
@@ -517,88 +519,87 @@ export class Wallet {
   }
 
   /**
-   * Consolidate unspents on a wallet
+   * Consolidate or fanout unspents on a wallet
+   *
+   * @param {String} routeName - either `consolidate` or `fanout`
    *
    * @param {Object} params - parameters object
+   *
+   * Wallet parameters:
    * @param {String} params.walletPassphrase - the users wallet passphrase
-   * @param {Number} params.limit - the number of unspents retrieved per call
-   * @param {Number} params.minValue - the minimum value of unspents to use in satoshis/kB
-   * @param {Number} params.maxValue - the maximum value of unspents to use in satoshis/kB
-   * @param {Number} params.minHeight - the minimum height of unspents on the block chain to use
-   * @param {Number} params.minConfirms - all selected unspents will have at least this many confirmations
+   * @param {String} params.xprv - the private key in string form if the walletPassphrase is not available
+   *
+   * Fee parameters:
    * @param {Number} params.feeRate - The fee rate to use for the consolidation in satoshis/kB
    * @param {Number} params.maxFeeRate - upper limit for feeRate in satoshis/kB
    * @param {Number} params.maxFeePercentage - the maximum relative portion that you're willing to spend towards fees
+   * @param {Number} params.feeTxConfirmTarget - estimate the fees to aim for first confirmation with this number of blocks
+   *
+   * Input parameters:
+   * @param {Number} params.minValue - the minimum value of unspents to use in satoshis
+   * @param {Number} params.maxValue - the maximum value of unspents to use in satoshis
+   * @param {Number} params.minHeight - the minimum height of unspents on the block chain to use
+   * @param {Number} params.minConfirms - all selected unspents will have at least this many confirmations
+   * @param {Boolean} params.enforceMinConfirmsForChange - if true, minConfirms also applies to change outputs
+   * @param {Number} params.limit                for routeName === 'consolidate'
+   *                 params.maxNumUnspentsToUse  for routeName === 'fanout'
+   *                  - maximum number of unspents you want to use in the transaction
+   * Output parameters:
+   * @param {Number} params.numUnspentsToMake - the number of new unspents to make
```

Comment:
I believe that this defaults to 1 at least for consolidate. That should perhaps be mentioned here.

Reply:
Not sure about that -- all these parameters have defaults that are defined on platform. I think devs should look it up there instead of always repeating it. 

I updated the docs here just for consistency, I actually don't think this is the right place for the full documentation.